health/health.d/systemdunits.conf

```diff
@@ -0,0 +1,112 @@
+## Check if the are any systemd units in the failed state (crashed).
+## States: 1 - active, 2 - inactive, 3 - activating, 4 - deactivating, 5 - failed.
+
+## Service units
+template: systemd_service_units_state
+      on: systemd.service_units_state
+  lookup: max -1s min2max
```

Comment:
> max -1s

Intention is to get the latest value (current state).

Read it as: 

> Notify if there are any units in the failed state now

Since we `every: 10s` it is possible to miss a unit that crashed, but was restarted by systemd during `every`. 

It is _by design_.